docs/line.rst

```diff
@@ -316,6 +316,20 @@ Line Methods
             If a shape is passed it must be an actual single shape object. It cannot be a
             tuple or list of coordinates that represent the shape. This is because there
             is no way to determine what type of shape the coordinates represent.
+        
+      .. ## Line.collideswith ##
+
+
+    .. method:: as_circle
+
+        | :sl:`instantiates a circle from a line`
```

Comment:
maybe change this a bit?

Reply:
Could you elaborate?

Reply:
It's a bit misleading, maybe something along this line "returns the smallest circle that fully encloses the line".